src/BuiltinExtensions/ComfyUIBackend/ExtraNodes/SwarmComfyExtra/SwarmYolo.py

```diff
@@ -10,14 +10,15 @@ def INPUT_TYPES(cls):
                 "image": ("IMAGE",),
                 "model_name": (folder_paths.get_filename_list("yolov8"), ),
                 "index": ("INT", { "default": 0, "min": 0, "max": 256, "step": 1 }),
+                "class_filter": ("STRING", { "default": "", "multiline": False }),
             },
         }
 
     CATEGORY = "SwarmUI/masks"
     RETURN_TYPES = ("MASK",)
     FUNCTION = "seg"
 
-    def seg(self, image, model_name, index):
+    def seg(self, image, model_name, index, class_filter):
```

Comment:
add a `= None` and move the input type into `optional` instead of `required`. That way pre-existing workflows with the node don't break